administration/crd-extending-api-with-crds.adoc

```diff
@@ -0,0 +1,20 @@
+[id='crd-extending-api-with-crds']
+= Extending the Kubernetes API with Custom Resource Definitions
+{product-author}
```

Comment:
You need to include the common-attributes module and remove the duplicate entries.